source/core/index.ts

```diff
@@ -833,6 +844,15 @@ export default class Request extends Duplex implements RequestEvents<Request> {
 			}
 		}
 
+		// HTTPS options
+		if ('ca' in options) {
+			console.warn('WARNING: "options.ca" was never documented, please use "options.certificateAuthority"');
```

Comment:
Use `util.deprecate`

Reply:
According to Node.JS documentation `util.deprecrate` is made for functions.
https://nodejs.org/api/util.html#util_util_deprecate_fn_msg_code

Reply:
According to the Node.JS documentation `util.deprecrate` is made for functions.
https://nodejs.org/api/util.html#util_util_deprecate_fn_msg_code